hazelcast/src/main/java/com/hazelcast/config/alto/AltoConfig.java

```diff
@@ -25,11 +22,7 @@ public int getEventloopCount() {
     }
 
     public AltoConfig setEventloopCount(int eventloopCount) {
```

Comment:
I believe every method should have a documentation, along with the since tag

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/57f4a326887f98dfdb6329505b31d651c2376e7c